examples/terraform-aws-lambda-example/main.tf

```diff
@@ -0,0 +1,39 @@
+# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
Ah I realized you are using tf12 syntax. We haven't updated our CI routine to use tf12 yet so this will probably fail. We have started the work though so will likely have a PR with the updates in the next week. Probably makes sense to do that instead of updating to tf11.